Makefile

```diff
@@ -15,6 +15,8 @@ dev-basic: build check basic-test
 BUILD_FLAGS ?=
 BUILD_TAGS ?=
 BUILD_CGO_ENABLED := 0
+BUILD_TOOL_CGO_ENABLED := 0
+BUILD_EXPERIMENT ?=
```

Comment:
IMO, the name is too general.

Reply:
It is used for GOEXPERIMENT, can u give some suggestions？

Reply:
got it, how about `BUILD_GOEXPERIMENT`